### ISSUE TITLE
Too many warnings hinders save

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#10516]
### Description

OK, we made an edit, pressed Save, entered our changeset comment, and...
Let's say there are 1000 warnings. I.e., the scrollbar indicator "A" is even smaller.
And say those warnings are all for bus routes and we have never rode a bus in our lives and they certainly are not our fault.

So we want to hurry up and find the final Upload button, but we must scroll past all those warnings first.

Yes we could grab the scrollbar and force it down.

But we don't edit often and are not sure that there is even a final Upload button at the end of all those warnings.

We try the X (B) but that just cancels our edit!

One solution would be a floating final Upload button, always visible.

### Screenshots

![Screenshot 2024-10-26 17 31 38](https://github.com/user-attachments/assets/3073383b-9246-4cb0-a8b0-b1e4c2a216ea)

**Comment by StC-OSM** [comment]
I was going to create a ticket to complain about missing warnings when I found this one 😄 

I believe that the real issue here is the design choice to display all warnings, including those that were there previously. Ideally there would be an option to see them, but the default behavior would be to display only warnings triggered by the changeset that is being uploaded

**Comment by andrewharvey** [comment]
> Ideally there would be an option to see them, but the default behavior would be to display only warnings triggered by the changeset that is being uploaded

Agreed, I've seen users completely new to OSM, feel overwhelmed and confused by these warnings unrelated to their changes.

**Comment by tyrasd** [comment]
Usually, iD only shows errors for features one has actually modified in the changeset (incl. newly created features). But especially for route relations, it can easily happen that a small change in a single road indirectly touches potentially lots of relations.

I agree that the UI is in such situations totally overloaded with warnings and not useful for the mappers. A simple solution would be to limit the number of displayed warnings and have an ellipsis at the end to indicate that there are actually more warnings.

**Comment by StC-OSM** [comment]
As discussed in [another issue report](https://github.com/openstreetmap/iD/issues/10530), an admittedly less simple solution would be to only display the warnings that were not present before the changeset (i.e. not burden the contributor with the errors of their predecessors). Less simple, but more accurate. Especially in terms of route breaking.

**Comment by Ravish990** [comment]
Hi, My name is Ravish . I study at Polaris School Of Technology ,where I am in first Year. I am interest in contributing the organization and solving this issue. Will you please assign me the issue?? I love to fix it properly.

**Comment by tyrasd** [comment]
this issue is already being worked on by @AkashNegi1 in #10595

**Comment by Ravish990** [comment]
Ok I am unaware about it.
Any help feel free to reach out I am always excited for this.

**Comment by kanavbajaj** [comment]
Can I work on this issue?

**Comment by k-yle** [comment]
I removed https://github.com/openstreetmap/iD/labels/new%20contributor%20opportunity because I don't think hiding genuine warnings is the right solution ([#10595](https://github.com/openstreetmap/iD/pull/10595)). 


Instead, we could either:
- make use of the new system for 'low severity' warnings ([#11020](https://github.com/openstreetmap/iD/pull/11020))
- refine the logic which determines whether a feature was modified. For example, if I'm just realigning a road, then I didn't actually edit the bus routes. Therefore, I shouldn't see warnings about those bus routes on the changeset page. This is the example from the original issue.
